src/FSharpLint.Core/Text.resx

```diff
@@ -174,6 +174,9 @@
   <data name="RulesNamingConventionsUnderscoreError" xml:space="preserve">
     <value>Consider changing `{0}` to remove any underscores.</value>
   </data>
+  <data name="RulesVariableLengthGreaterThanOneError" xml:space="preserve">
+    <value>Consider changing `{0}` to more than 1 letter.</value>
```

Comment:
let's use the word "single" instead of the number "1"

Reply:
-> `than a single letter`

Reply:
Updated.